src/ucp/wireup/wireup_cm.c

```diff
@@ -1011,6 +1011,16 @@ ucp_ep_cm_server_create_connected(ucp_worker_h worker, unsigned ep_init_flags,
                                                    conn_request->dev_name);
     ucp_ep_h ep;
     ucs_status_t status;
+    char client_addr_str[UCS_SOCKADDR_STRING_LEN];
+
+    if (tl_bitmap == 0) {
+        ucs_error("listener %p: got connection request from %s on uninitialized device %s",
```

Comment:
what does this mean that the device was not initialized?
what should the user do in case he sees this error?

Reply:
> what does this mean that the device was not initialized?

For example, port was down and UCX did not initialize TLs for this device

> what should the user do in case he sees this error?

restart epp to reinitialize UCX or ignore taking into account that all clients which will connect to this device cannot be handled.

Reply:
thanks for clarifying!

IMHO 'uninitialized' should be 'an unusable' then.

Reply:
from UCX perspective it's "uninitialized", @yosefe WDYT?

Reply:
need to find something more meaningful for users, can you suggest?

Reply:
would the user know what "uninitialized" means and what to do with it?
IMHO 'unusable' gives more of indication of what the user should do and what is wrong.

maybe something like:
"listener %p: got connection request from %s on an usable device %s. Please check the device state"

although, if the user checks the state of the port and it's up but was down before, the message would be more confusing than helpful.

Reply:
maybe "got connection request on a device which was not present during UCP initialization

Reply:
👍

Reply:
updated